src/targets/services/aggregation.js

```diff
@@ -42,7 +42,7 @@ export const aggregation = async () => {
   // Fetch all stored shares
   const compressedShares = []
   for (let s of receivedShares) {
-    // TODO: Should use cozy-client, but fetchFileContentById uses fetch instead of fetchJSON
+    // TODO: Should use cozy-client, but fetchFileContentById returns a XMLHttpRequest that fails when parsed
```

Comment:
But why does it fail? :smile: 
`fetchJSON` also uses `fetch`. Maybe it's related to the content-type? https://github.com/cozy/cozy-client/blob/1ab42dff2e9088f4a3daff29866ad7a22b093791/packages/cozy-stack-client/src/CozyStackClient.js#L274